templates/firm/show.json.twig

```diff
@@ -0,0 +1,61 @@
+{
+    "@context": "https://schema.org",
+    "@graph": [
+        {
+            "@type": "Organization"
+            {% if firm.name %},"name": "{{firm.name}}" {% endif %}
+            {% if firm.startDate %}, "foundingDate": "{{ firm.startDate }}" {% endif %}
+            {% if firm.endDate %} ,"dissolutionDate": "{{ firm.endDate }}" {% endif %},
+            "@id": "{{ url('firm_show', {'id':firm.id}) }}",
+            "address": {
+                "@type": "PostalAddress"
+                {% if firm.city %},"addressLocality": "{{ firm.city }}" {% endif %}
+                {% if firm.streetAddress %}, "streetAddress": "{{ firm.streetAddress }}" {% endif %}
+            }
+        }
+        {% for titleFirmRole in firm.getTitleFirmRoles("Publisher") %}
```

Comment:
getTitleFirmroles() takes a boolean as a parameter and will sort if that boolean is true. Passing a string isn't right and it'll break in a future PHP version where things are more strict.